core/src/main/java/sunstone/core/SunstoneConfig.java

```diff
@@ -0,0 +1,79 @@
+package sunstone.core;
+
+
+import io.smallrye.config.SmallRyeConfig;
+import io.smallrye.config.SmallRyeConfigBuilder;
+import org.eclipse.microprofile.config.spi.ConfigSource;
+
+import java.util.Objects;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+public class SunstoneConfig {
+
+    static final SmallRyeConfig config = new SmallRyeConfigBuilder()
+            .addDefaultInterceptors()
+            .addDefaultSources()
+            .withSources(new SunstoneExpressionSource())
+            .build();
+
+    public static String getString(String key) {
```

Comment:
Maybe add some javadoc as this is main usage.

Reply:
Added.